code/datums/mutations.dm

```diff
@@ -282,6 +282,7 @@
 	if(..())
 		return
 	owner.resize = 0.8
+	owner.ventcrawler = 1
```

Comment:
Yes it can and in fact TG broke ventcrawl 1 because the item check doesn't discriminate organs, I'm fixing this shit as we speak